src/payments.ts

```diff
@@ -148,6 +149,24 @@ export class Payments {
     return !!this.nvmApiKey
   }
 
+  /**
+   * Account address
+   *
+   * @example
+   * ```
+   * payments.accountAddress
+   * ```
+   *
+   * @returns Account address when the user is logged in.
+   */
+  get accountAddress(): string | undefined {
+    if (!this.nvmApiKey) {
+      return undefined
+    }
+
+    return decodeJwt(this.nvmApiKey).iss
```

Comment:
I think the `iss` is part of the `marketplaceAuthToken` but not the `nvpApiKey`. Meaning it should be something like this:
```
decodeJwt(
    decodeJwt(this.nvmApiKey).marketplaceAuthToken
    ).iss
```